### PR TITLE
MGMT-11255: Update the Bugzilla plugin configuration for all the assisted repositories

### PR DESCRIPTION
**Pull request by eliorerz**
Update the Bugzilla plugin configuration for all the assisted repositories repositories to include the release branches 2.6, 2.7 and 2.8.

/cc @osherdp

### CODE REVIEW

File: core-services/prow/02_config/openshift/cluster-api-provider-agent/_pluginconfig.yaml [hunk 1]
```diff
@@ -25,9 +25,22 @@ bugzilla:
               - rhacm-2.6
               target_release: rhacm-2.5
             release-ocm-2.6:
+              dependent_bug_states:
```
Comment (eliorerz) [inline core-services/prow/02_config/openshift/cluster-api-provider-agent/_pluginconfig.yaml:28]: FYI @eranco74

File: core-services/prow/02_config/openshift/cluster-api-provider-agent/_pluginconfig.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ bugzilla:
         cluster-api-provider-agent:
           branches:
             master:
-              target_release: rhacm-2.6
+              target_release: rhacm-2.7
```
Comment (eliorerz) [inline core-services/prow/02_config/openshift/cluster-api-provider-agent/_pluginconfig.yaml:13]: FYI @eranco74

File: core-services/prow/02_config/openshift/cluster-api-provider-agent/_pluginconfig.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ bugzilla:
         cluster-api-provider-agent:
```
Comment (eliorerz) [inline core-services/prow/02_config/openshift/cluster-api-provider-agent/_pluginconfig.yaml:10]: FYI @eranco74

**Review by osherdp — APPROVED** [review]

**Pull request merged**